src/utils/Contracts.js

```diff
@@ -14,7 +14,11 @@ const DEFAULT_COVERAGE_TX_PARAMS = {
 
 export default {
   getFromLocal(contractName) {
-    const buildDir = `${process.cwd()}/build/contracts`
+    let buildDir = `${process.cwd()}/build/contracts`
+    const truffleConfig = require(`${process.cwd()}/truffle.js`)
```

Comment:
Instead of manually loading the truffle config file, let's better use [truffle-config](https://github.com/trufflesuite/truffle/blob/develop/packages/truffle-config/index.js), as it already handles some nuances (such as falling back to `truffle-config.js` if `truffle.js` is missing). There is already an example for this [in CLI](https://github.com/zeppelinos/zos-cli/blob/9c4ac25d90204d173d09216dcee0955fd14d6fbb/src/models/truffle/Truffle.js#L9-L10).